src/library/itunes/ituneslocalhosttoken.cpp

```diff
@@ -0,0 +1,11 @@
+#include "library/itunes/ituneslocalhosttoken.h"
+
+#include <QString>
+
+QString iTunesLocalhostToken() {
+#if defined(__WINDOWS__)
+    return "//localhost/";
+#else
+    return "//localhost";
+#endif
+}
```

Comment:
this could just be a `static const(expr)` in the header, right?

```cpp
static const QString kiTunesLocalHostToken = QStringLiteral("//localhost"
#if defined(__WINDOWS__)
"/"
#endif
);
```